cellfinder/core/main.py

```diff
@@ -54,6 +55,11 @@ def main(
         Called with the batch number that has just finished.
     detect_finished_callback : Callable[list], optional
         Called after detection is finished with the list of detected points.
+        Number of points processed simultaneously by the neural network.
```

Comment:
Should this be after "batch_size:"?
Or I don't understand this change?

Reply:
Also, I think it's "number of planes copied to the GPU memory at once"? There is no neural network used during detection.